app/views/content/is-teaching-right-for-me/teacher-pay-and-benefits.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: "Teacher pay and benefits"
 heading: "Teacher pay and benefits"
+subcategory: Pay and benefits
 description: |-
   All qualified teachers will have a starting salary of at least £30,000 (or higher in London). Find out about teacher pay ranges and more benefits of teaching.
 date: "2021-06-24"
-image: "static/content/hero-images/0008.jpg"
 backlink: "../../"
-navigation: 25
+navigation: 5.10
+navigation_title: How much do teachers get paid?
+navigation_description: Find out the starting salary for a qualified teacher and how this can increase during your career.
```

Comment:
Not sure this is quite right at the moment - it implies that the starting salary increases!

Do improve my suggestion as needed though

```suggestion
navigation_description: Find out the starting salary for a qualified teacher and how your pay can increase during your career.
```